pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -141,6 +141,13 @@ class StateVectorCudaManaged
 
     ~StateVectorCudaManaged() = default;
 
+    /**
+     * @brief Get the total number of wires.
+     */
+    auto getTotalNumQubits() const -> size_t {
```

Comment:
What's the difference between `getNumQubits` and `getTotalNumQubits`? Do we necessarily need to define this method to the API?